src/els_command_ct_run_test.erl

```diff
@@ -46,8 +47,8 @@ publish_result(Uri, Line, Severity, Message) ->
   els_diagnostics_provider:publish(Uri, [D]),
   ok.
 
--spec run_test(atom(), atom()) -> any().
+-spec run_test(atom(), atom()) -> {ok, binary()} | {error, any()}.
```

Comment:
Should we add this contract in the documentation for the function configured to run tests in the remote node?

Reply:
I think we are going to have to come up with some clear specs/guidelines/protocols for how the LS node interacts with the runtime one.

Can that be defined via bahaviours?